addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2972,7 +2967,17 @@ var SnippetsMenu = Widget.extend({
      * @private
      * @param {jQuery}
      */
-    _patchForComputeSnippetTemplates($html) {},
+    _patchForComputeSnippetTemplates($html) {
+        // TODO: remove in master. Updates icons and images in non-editable
+        // environments to be editable thanks to the o_editable_media class
+        const addVXML = function(index, elem) { 
+            elem.dataset.vxml = "001";
+        };
+        $html.find('div.s_social_media').each(addVXML);
+        $html.find('section.s_company_team').each(addVXML);
+        $html.find('div.s_social_media').find('i').each(function (index, elem) { elem.classList.add('o_editable_media') });
+        $html.find('section.s_company_team').find('img').addClass('o_editable_media');
```

Comment:
@dmo-odoo @qsm-odoo The patch correctly works for adding the `data-vxml` attribute, which indeed displays a warning about an outdated block even in the case where the app hasn't been updated.
The patch does _not_ work to add the `o_editable_media` class. Both ways of adding the class do affect the `$html` but not the actual contents of the page I'm editing. So I'd like some pointers on that.

I'm also a bit confused about the following (assuming the aforementioned issue is fixed):
- if the user does _not_ update, what's the purpose of the outdated warning since we "forcibly" added the class ?
- if the user _does_ update, the XML will be updated and thus the snippets in the "hardcoded" demo website and the templates for adding new snippets will be updated along with it. So I assume the warning is there only for "old" snippets previously added by the user. But won't the patch affect those exactly ?

Sorry for all the confusion.